webpack.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = (env = {}) => {
     new InlineEnvironmentVariablesPlugin('SCRIVITO_TENANT'),
     new CopyWebpackPlugin([
       { from: '../public' },
+      { from: '../src/intercom.js' },
```

Comment:
Why a copy, and not something like this https://github.com/Scrivito/scrivito_example_app_js/blob/master/webpack.config.js#L74 ?

Reply:
Because the intercom.js includes only the snippet copied from the intercom documentation in my opinion it makes no sense to let it run through babel. For google_analytics.js it is an other case, we compile it because of the `import` statement in it. 

I've tried it. The size of the intercom.js gets doubled if we include it as own entry in webpack (533B -> 981B).